server/nuxeo-nxr-server/src/main/resources/templates/docker/lib/log4j2.xml

```diff
@@ -192,6 +192,8 @@
     <Logger name="org.apache.shindig.common.xml.XmlUtil" level="warn" />
     <Logger name="org.apache.myfaces.shared_tomahawk" level="warn" />
     <Logger name="net.openhft.chronicle.queue.impl.single.SCQIndexing" level="error" /> <!-- Hide perf warnings -->
+    <Logger name="net.openhft.chronicle.bytes.MappedFile" level="error" /> <!-- Hide perf warnings -->
+    <Logger name="net.openhft.chronicle.queue.impl.single.ReferenceCountedCache" level="error" /> <!-- Hide cache cleaning warnings -->
```

Comment:
Shouldn't we put this in the common log4j2.xml file?

Reply:
indeed I only updated the docker log4j2 conf I will handle the default log4j2